app/src/main/res/layout/activity_events.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</RelativeLayout>
```

Comment:
New line

Reply:
It's meant to be empty because I haven't added a recyclerview yet